jlama-core/src/main/java/com/github/tjake/jlama/util/Downloader.java

```diff
@@ -55,7 +55,7 @@ public Downloader(String modelDir, String model) {
 
     }
 
-    public Downloader notDownloadWieghts() {
+    public Downloader notDownloadWeights() {
```

Comment:
Thanks, maybe this should be called `skipWeights`?

Reply:
It's a good idea, and is very simple to understand.